.gitignore

```diff
@@ -14,3 +14,5 @@ composer.lock
 
 # PHPUnit cache
 .phpunit.result.cache
+
+/node_modules/
```

Comment:
should it not be `node_modules/` ? but i assume its ignoring properly

Reply:
it makes sure it's only top level - so if for some reason there would be a node_modules folder somewhere deeper it would not get ignored.